### PR TITLE
PIM-9750: Auto_increment reaches max value for pim_catalog_completeness table

### PR DESCRIPTION
**Pull request by tomglvng**

### CODE REVIEW

**Review by LaurentPetard — APPROVED** [review]

File: CHANGELOG-4.0.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 # 4.0.98 (2021-03-16)
 
+## Bug fixes
```
Comment (nmarniesse) [inline CHANGELOG-4.0.md:5]: Should be before the `# 4.0.98 (2021-03-16)` line

**Review by nmarniesse — APPROVED** [review]

File: upgrades/schema/Version_4_0_202103181422_update_pim_catalog_completeness_id.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+namespace Pim\Upgrade\schema;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * @copyright 2021 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+final class Version_4_0_202103181422_update_pim_catalog_completeness_id extends AbstractMigration
```
Comment (BitOne) [inline upgrades/schema/Version_4_0_202103181422_update_pim_catalog_completeness_id.php:13]: No migration on released version.

**Review by BitOne — CHANGES_REQUESTED** [review]
How to make sure this PR will

File: upgrades/schema/Version_4_0_202103181422_update_pim_catalog_completeness_id.php (hunk 2 above)
Comment (fitn) [reply upgrades/schema/Version_4_0_202103181422_update_pim_catalog_completeness_id.php:13]: @BitOne Please suggest a solution

**Review by BitOne — APPROVED** [review]

**Pull request merged**